src/transformers/configuration_utils.py

```diff
@@ -236,6 +236,9 @@ class PretrainedConfig(PushToHubMixin):
 
         use_bfloat16 (`bool`, *optional*, defaults to `False`):
             Whether or not the model should use BFloat16 scalars (only used by some TensorFlow models).
+        tf_legacy_loss (`bool`, *optional*, defaults to `False`):
+            Whether or not the model should use legacy TensorFlow losses. Legacy losses have variable output
+            shapes and may not be XLA-compatible.
```

Comment:
Maybe add that this parameter is here for backward compatibility but will be removed in v5?

Reply:
Done!